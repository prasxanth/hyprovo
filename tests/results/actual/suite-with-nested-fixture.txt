2021-11-16 09:27:35.623 - INFO - Pass ✓... (suite-with-nested-fixture / test-add-variable): add pass
2021-11-16 09:27:35.625 - ERROR - Fail ✗... (suite-with-nested-fixture / test-add-variable): add fail 
                                    Expected: 3
                                    Got: 2
2021-11-16 09:27:35.630 - INFO - Pass ✓... (suite-with-nested-fixture / test-mul-variable): mul pass
2021-11-16 09:27:35.632 - ERROR - Fail ✗... (suite-with-nested-fixture / test-mul-variable): mul fail 
                                    Expected: 4
                                    Got: 1
