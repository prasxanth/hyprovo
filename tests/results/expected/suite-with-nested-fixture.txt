2021-11-14 11:26:50.314 - INFO - Pass ✓... (suite-with-nested-fixture / test-add-variable): add pass
2021-11-14 11:26:50.319 - ERROR - Fail ✗... (suite-with-nested-fixture / test-add-variable): add fail 
                                    Expected: 3
                                    Got: 2
2021-11-14 11:26:50.327 - INFO - Pass ✓... (suite-with-nested-fixture / test-mul-variable): mul pass
2021-11-14 11:26:50.332 - ERROR - Fail ✗... (suite-with-nested-fixture / test-mul-variable): mul fail 
                                    Expected: 4
                                    Got: 1
